lib/IRGen/IRGen.cpp

```diff
@@ -148,6 +148,8 @@ swift::getIRTargetOptions(const IRGenOptions &Opts, ASTContext &Ctx) {
   if (Clang->getTargetInfo().getTriple().isOSBinFormatWasm())
     TargetOpts.ThreadModel = llvm::ThreadModel::Single;
 
+  TargetOpts.EmulatedTLS = Opts.UseEmulatedTLS;
```

Comment:
As @compnerd suggested, maybe we should copy the setting from Clang's options? A simple way might be do to something like this (untested, written in this comment):

```suggestion
  TargetOpts.EmulatedTLS = Opts.UseEmulatedTLS || Clang->getCodeGenOpts().EmulatedTLS;
```

Reply:
As noted above yesterday, I already implemented support for that flag then. However, I just tried your suggestion and that works too and is simpler, so thanks for the suggestion: I will update this pull to reuse your suggested ClangImporter flag instead.

Reply:
Done